src/deps/zcl_abaplint_deps_find.clas.abap

```diff
@@ -217,7 +323,9 @@ CLASS ZCL_ABAPLINT_DEPS_FIND IMPLEMENTATION.
         CHANGING
           ct_tadir     = ct_tadir ).
     ELSE.
-      ASSERT 0 = 1.
+      FORMAT INTENSIFIED ON.
```

Comment:
where would you see the list output? we either ignore or raise an exception to pass it up to the ui

Reply:
I'm using the report ZABAPLINT_LIST_DEPS to test the dependencies.
The exception always kill the whole run and one does not know much more. There is no catch implemented either.
I'll change the output to be created only if it comes from this report.

How do you propose to handle the exception?